app/src/main/java/com/jerboa/db/entity/AppSettings.kt

```diff
@@ -24,6 +24,7 @@ data class AppSettings(
         defaultValue = "0",
     )
     val themeColor: Int,
+    // TODO get rid of this column next time you regenerate the app
```

Comment:
Why next time? why cant we drop it already?

Reply:
android SQLite can't drop columns, you have to regenerate the entire table. Should probably be done at some point, but probably when we go over all the columns and everything we need to refactor.

Reply:
We have done this before, see when I changed the default of a column